webdriver-factory/pom.xml

```diff
@@ -41,6 +41,63 @@
         </plugins>
     </build>
 
+    <dependencyManagement>
```

Comment:
This section should reside in the parent pom. Any references to versions of artifacts should be centralised in the parent.